media_driver/agnostic/common/codec/hal/codechal_encode_vp8.cpp

```diff
@@ -3813,8 +3813,7 @@ MOS_STATUS CodechalEncodeVp8::ExecuteSliceLevel()
 
         if (m_brcEnabled)
         {
-            if (m_currPass == (m_numPasses - 1) ||
```

Comment:
the original code has logical error. It will set m_resSyncObjectVideoContextInUse at "m_currPass == (m_numPasses - 1)" and "m_currPass == (m_numPasses)", but it only check m_resSyncObjectVideoContextInUse at "m_currPass == (m_numPasses)"